### PR TITLE
bumbing up versions to remove vulnerabilites

### PR DESCRIPTION
**Pull request by bjornm82**
Bumping up versions to remove vulnerabilities

Currently on master:
Tested 171 dependencies for known issues, found 3 issues, 13 vulnerable paths.

This branch:
✓ Tested 221 dependencies for known issues, no vulnerable paths found.

TODO: Chart templates has some medium issues, won't be solved with this PR

### CODE REVIEW

File: Dockerfile
Comment (bnguyennhon) [inline Dockerfile:21]: I would prefer to pin this to the current latest version

Comment (bjornm82) [reply Dockerfile:21]: I normally also would take fixed versions. With this case it's used solely for getting the ca-certificates. From my point of view it would be better to assure always having the newest ca-cert.

I can't see any breaking issues when not using the latest, therefor I thought it might be better to use latest.

What do you think?

Comment (bjornm82) [reply Dockerfile:21]: I normally also would take fixed versions. With this case it's used solely for getting the ca-certificates. From my point of view it would be better to assure always having the newest ca-cert.

I can't see any breaking issues when not using the latest, therefor I thought it might be better to use latest.

What do you think?

Comment (volhakurylionak) [reply Dockerfile:21]: I dare vote for pinning, if something changes and breaks a year from now it will be harder to find, especially if it only runs to get a cert :/ IMHO I'd prefer the inconvenience of updating the pinned version every once in a while than having it misbehave after some restart and looking for this thing. Also maybe we should pull the image from our own ACR (put it there and pull from there) in case something is changed by docker hub again?

Comment (bjornm82) [reply Dockerfile:21]: Alright, so I'll pin it.

Comment (bjornm82) [reply Dockerfile]: Done, is now 3.13

**Review by volhakurylionak — APPROVED** [review]

File: Dockerfile
Comment (bnguyennhon) [inline Dockerfile:23]: could it be possible to also create & use a non-root user for the runtime image ?

**Review by csimplestring — APPROVED** [review]

**Review by nielstenboom — APPROVED** [review]
LGTM 👍  , let's see if the full integration tests run after it's merged: https://ci.freemium.diw.rtlwecloudservices.nl/teams/main/pipelines/phoenix-dev/jobs/test-build-and-push-image/builds/2

File: Dockerfile
Comment (bjornm82) [reply Dockerfile:23]: I was under the assumption we were not running with Alpine, seems weird we don't actually run a scratch image. You know the reason @nielstenboom ?

Comment (nielstenboom) [reply Dockerfile:23]: No clue, all Davide's work from a long while ago

Comment (bjornm82) [reply Dockerfile:23]: Would it make sense to move to a scratch image in stead of alpine? I think it's overkill to use an OS for Go.

Comment (csimplestring) [reply Dockerfile:23]: make sense to me. A Scratch image is more secured at least (you have no extra deps)

Comment (bnguyennhon) [reply Dockerfile:23]: sounds good 👍

**Pull request merged**
